### ISSUE TITLE
Example Notebook/script to access forced sources parameters from the database. 

### ISSUE THREAD

**Issue by rbiswas4** [DarkEnergyScienceCollaboration/Monitor#37]
@jchiang87 I am looking for an example of how we can find parameters relating to all point like forced sources in the database. I would love to start with SN only, but would be OK if other classes of objects were there as well. I am trying to figure out how to generally associate objects quickly between the reference and outputs of DM.

**Comment by jchiang87** [comment]
@rbiswas4 Which forced source parameters were you interested in searching on?  For the comparisons I made in #29, I'm just doing a positional query on RA, Dec, e.g., [here](https://github.com/jchiang87/desc_projects/blob/master/phosimTables/PhosimDbInterface.py#L56).

**Comment by rbiswas4** [comment]
Yes, that is the kind of thing I would like, let me look more carefully after the meeting:
-  I wanted to get RA, Dec, sequence of MJD where they were measured to have 'significant' non zero fluxes, or SNR values.  
- I want to iterate over all such objects with various thresholds. So what I am really looking for is the sequence of ID for  all forced sources.
- How do you figure out if they are like SN and not AGN for example?

**Comment by jchiang87** [comment]
> How do you figure out if they are like SN and not AGN for example?

This might be a useful thing to discuss during the meeting.  If you histogram the peak normalized light curves from the Level 2 data, the distributions clearly show three families: constant sources, AGNs and SNe.  The AGNs tend to have a broad distribution of fluxes while the SNe have mostly quiescent values and a tail of higher fluxes.  (There is actually a fourth class: bright constant sources near the edges of the sensors from some visits.  These have a tail of lower flux values.  I think this is an artifact of our single sensor runs and/or phosim.)

**Comment by rbiswas4** [comment]
Ok. When you say broad distribution of fluxes do you really mean that AGN are long lived and SN are short lived so most of the SN fluxes are consistent with 0 ? That is true and an easy way to distinguish over long time periods.
